main.js

```diff
@@ -437,7 +458,6 @@
         console.log('All data loaded');
     }).catch((err) => {
         console.log('final error catch data', err);
```

Comment:
How would you do this, just make Promise a variable and call it? Sorry lol new to JS